CMakeLists.txt

```
@@ -346,14 +338,13 @@ if(Launcher_QT_VERSION_MAJOR EQUAL 6)
 endif()
 
 if(NOT Launcher_FORCE_BUNDLED_LIBS)
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(libarchive IMPORTED_TARGET libarchive)
```

This should also be taken out of the current conditional since no bundled version of this exists
```suggestion
    find_package(PkgConfig REQUIRED)
    pkg_check_modules(libarchive REQUIRED IMPORTED_TARGET libarchive)
```